### ISSUE TITLE
Missing tooltip for VPN icon

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#22626]
## Description 

Missing tooltip for VPN icon

## Steps to Reproduce

   1. install `1.39.86`
   2. launch Brave
   3. set `brave://flags/#brave-vpn` to `Enabled`
   4. click on `Relaunch`
   5. hover over the VPN icon in the browser toolbar

## Actual result:

Nothing happens - no tooltip on hover

<img width="1312" alt="Screen Shot 2022-04-29 at 10 05 33 AM" src="https://user-images.githubusercontent.com/387249/165991218-f002b10f-84fd-40fd-9ae0-91e7acb0efa2.png">

## Expected result:

Tooltip - either static or dynamic (based on VPN-connection state)

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.39.86 Chromium: 101.0.4951.41 (Official Build) beta (x86_64)
-- | --
Revision | 93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}
OS | macOS Version 11.6.5 (Build 20G527)

@aguscruiz @rebron

**Comment by kjozwiak** [comment]
The above requires `1.46.83` or higher for `1.46.x` verification 👍

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.46.86 Chromium: 107.0.5304.91 (Official Build) beta (x86_64)
-- | --
Revision | 3d5948960d62418160796d5831a4d2d7d6c90fa8-refs/branch-heads/5304@{#1097}
OS | macOS Version 11.7.1 (Build 20G918)

### Steps:
1. installed `1.46.86`
2. launched Brave
3. opened `brave://flags`
4. set `SDK` and `VPN` flags to `Enabled`
5. clicked `Relaunch`
6. loaded `account.bravesoftware.com`
7. purchased `Brave Firewall & VPN`
8. clicked on `Refresh Brave VPN`
9. hovered over the `VPN` button on the browser toolbar
10. confirmed it read `Brave Firewall & VPN: Disconnected`
11. connected to VPN
12. hovered over the VPN button on the browser toolbar
13. confirmed it read `Brave Firewall & VPN: Connected`

`disconnected` | `connected`
--------------|-----------
<img width="1211" alt="Screen Shot 2022-10-31 at 9 31 03 AM" src="https://user-images.githubusercontent.com/387249/199061148-2973e786-aeab-498e-9944-4d0411e23fe3.png"> | <img width="1213" alt="Screen Shot 2022-10-31 at 9 31 21 AM" src="https://user-images.githubusercontent.com/387249/199061161-6332b9af-b604-4009-9017-0a697766d9b5.png">
